lb_content_resolver/unresolved_recording.py

```diff
@@ -42,13 +42,33 @@ def add(self, recording_mbids):
             of times it has been unresolved.
         """
 
-        query = """INSERT INTO unresolved_recording (recording_mbid, last_updated, lookup_count)
-                        VALUES (?, ?, 1)
-         ON CONFLICT DO UPDATE SET lookup_count = EXCLUDED.lookup_count + 1"""
+        recording_mbids = list(set(recording_mbids))
+
+        placeholders = ",".join(("?", ) * len(recording_mbids))
+        existing = {
+                row[0]:True for row in db.execute_sql(
+                """SELECT recording_mbid
+                     FROM unresolved_recording
+                    WHERE recording_mbid IN (%s)""" %
+                placeholders, tuple(recording_mbids)).fetchall()
+        }
 
         with db.atomic() as transaction:
```

Comment:
```suggestion
        with db.atomic():
```

`transaction` never used.